Docs/Design/Meeting_Intelligence_API.md

```
@@ -0,0 +1,122 @@
+Sharing:
+- `POST /api/v1/meetings/sessions/{session_id}/share/slack`
+- `POST /api/v1/meetings/sessions/{session_id}/share/webhook`
+
+Live transport:
+- `GET /api/v1/meetings/sessions/{session_id}/events` (SSE)
+- `WS /api/v1/meetings/sessions/{session_id}/stream`
+
+## Key Request Models
+
+Create session:
+```json
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)